src/TritonCTS/src/DbWrapper.cpp

```diff
@@ -234,34 +234,35 @@ void DbWrapper::writeClockNetsToDb(Clock& clockNet) {
         std::map<int,uint> fanoutcount;
 
         // create subNets
-        unsigned numClkNets = 0; 
+        _numClkNets = 0; 
         const Clock::SubNet* rootSubNet = nullptr;
         clockNet.forEachSubNet( [&] (const Clock::SubNet& subNet) {
                         bool outputPinFound = true;
                         bool inputPinFound = true;
+                        bool leafLevelNet = false;
                         //std::cout << "    SubNet: " << subNet.getName() << "\n";
                         if (("clknet_0_" + clockNet.getName()) == subNet.getName()){
                                 rootSubNet = &subNet;
                         }
                         odb::dbNet* clkSubNet = odb::dbNet::create(_block, subNet.getName().c_str());
-                        ++numClkNets;
+                        ++_numClkNets;
                         clkSubNet->setSigType(odb::dbSigType::CLOCK);
                         
                         //std::cout << "      Driver: " << subNet.getDriver()->getName() << "\n";
                        
                         odb::dbInst* driver = _block->findInst(subNet.getDriver()->getName().c_str());
```

Comment:
why doesn't the driver have a pointer to the dbInst so it doesn't have to do a name lookup here?

Reply:
I've added a simple modification so that dbInsts references are added to the ClockInst (TritonCTS object) during the buffer creation instead.